test/fixtures/component-playground/proxies.js

```diff
@@ -29,6 +29,12 @@ const MarkupProxy = React.createClass({
 });
 
 module.exports = _.merge({}, selectedFixture, {
-  proxies: [PropMutatorProxy, MarkupProxy],
+  proxies: [{
+    component: PropMutatorProxy,
+    props: {},
```

Comment:
OK. I got it now, this is a fixture for CP, not one used by CP. My bad.

That being said, this means you can only send the contents of the store once to the ReduxProxy, global for all component/fixtures loaded? Is this correct?

If yes, I suggest the following alternative: Revert the props thing and continue to send the entire fixture to all proxies. This way, ReduxProxy continues to reads the contents of the store from `props.store`, but `store` will be a separate property on every fixture. Wdyt?